src/main/java/seedu/bankwithus/BankWithUs.java

```diff
@@ -3,13 +3,18 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 
+import seedu.bankwithus.exceptions.CommandNotFoundException;
+
 public class BankWithUs {
 
     public static final String FILE_PATH = "data/save.txt";
 
+    public boolean isExitEntered = false;
```

Comment:
Cant think of a better name than this lol. I really wanted to just omit the stupid is, but... coding standard violation...